packages/office-addin-manifest/src/export.ts

```diff
@@ -31,7 +38,36 @@ async function createZip(manifestPath: string): Promise<AdmZip> {
     throw new Error(`The file '${manifestPath}' does not exist`);
   }
 
-  const manifest: devPreview.DevPreviewSchema = await ManifestUtil.loadFromPath(manifestPath);
+  const manifest: DevPreviewSchema = readJsonFileSync(manifestPath);
+  const agents = manifest?.copilotAgents?.declarativeAgents;
+
+  if (agents) {
+    agents.forEach((agent, agentIndex) => {
+      const file: string = agent?.file;
+      const filePath = path.join(manifestDir, file);
+
+      if (!fs.existsSync(filePath)) {
+        throw new Error(
+          `copilotAgents.declarativeAgents[${agentIndex}].file does not exist:  ${filePath}`
+        );
+      }
+
+      zip.addLocalFile(filePath, "", file);
```

Comment:
What is the 3-argument version of this method?  I"ve only seen the 1 or 2 argument version and that is all that is documented (which isn't great).

Reply:
I was just following what I saw in code elsewhere. Not sure what the second parameter is but it seemed the first was the file and the third was the string name for the file.